templates/sios-datakeeper.template.yaml

```diff
@@ -1024,6 +1024,24 @@ Resources:
           - Name: InstanceId
             Selector: $.Reservations[0].Instances[0].InstanceId
             Type: String
+        - name: updateS3BucketName
+          action: aws:updateVariable
+          inputs:
+            Name: 'S3BucketName'
+            Value: 
+              - !If
+                - UsingDefaultBucket  # is this an AWS template? 
+                - !Sub '${QSS3BucketName}-${AWS::Region}' #true
+                - !Ref QSS3BucketName #false
+        - name: updateS3Region
+          action: aws:updateVariable
+          inputs:
+            Name: 'S3Region'
```

Comment:
Similar comment as above, S3Region should be QSS3BucketRegion (to match line 955).